### DISCUSSION TITLE
What VS Code Version ?

### DISCUSSION THREAD

**Discussion by vsjha18** [coder/code-server#2176] [Q&A] [answered]
I develop private VS Code extensions for my company. It is important for me to understand which version of `code-server` corresponds to which version of `vs code`. This information will be useful for me so that I can understand which API's would be available in `code-server` in case we have to use some bleeding edge APIs.

Also how much generally is the lag between a version of `code-server` and `vscode`.

**Comment by code-asher** [comment, marked as answer]
There are a couple places you can check the version of VS Code shipped
with a particular code-server release:

- The releases page on GitHub
- Run `code-server --version --json`
- Help > About from inside VS Code

The lag currently varies pretty widely but we're trying to get it down to
about a week at most. So far it's been around a month or more though.
